### ISSUE TITLE
tames - verb or noun?

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#123]
tames - verb or noun? created by palau371@gmail.com on 2017-02-15 14:33:02

**Comment by johnbent** [comment]
palau371@gmail.com replied, <br><br><p>today i've bought Japanese-Russian dictionary and now I find there a Japanese noun TAMESI that means "attempt". so, Japanese has the verb TAMESU and the noun TAMESI from the same root. they both could sound TAMES in Palauan.&nbsp;</p>

<p>please, can anybody provide 3-4 sentence examples with TAMES?</p>

**Comment by johnbent** [comment]
jlukesemiwo@gmail.com replied, <br><br>Tames - verb, at least the way we have adopted its use as "try" or "to
attempt." It's synonymous with "melasem"

1) Ak mo tames el leko ra party. I'll try to come to the party.

2) Kam tames er tial subong el kmo ng sebechem. Try these pants if they
fit.

3) Ng kmal meduch el meruul a kall meng diak lousbech ra chad el tames a
ikel loruul. She's an amazing cook that she doesn't even need anyone to try
the food she makes.

4) A rechad ra llach a mo tames el mesmechokl a rokui meng diak el bo ra
keras. The attorneys will attempt at a settlement to avoid trial.

I hope these help. I tried to use it in various contexts. Note, however,
that even though I said it's synonymous with "melasem" there are times when
one would work but not the other. For example, if you replaced "tames" with
"melasem" in sentences 1 and 4, they would still make sense and mean the
same thing, but not if you do with sentences 2 and 3.

J
